tests/scripts/curves.pl

```diff
@@ -41,6 +64,47 @@ sub abort {
     exit 1;
 }
 
+# Disable all the curves. We'll then re-enable them one by one.
+for my $curve (@curves) {
+    system( "scripts/config.pl unset $curve" )
+        and abort "Failed to disable $curve\n";
+}
+# Depends on a specific curve. Also, ignore error if it wasn't enabled.
+system( "scripts/config.pl unset MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED" );
+
+# Test with only $curve enabled, for each $curve.
+for my $curve (@curves) {
+    system( "make clean" ) and die;
+
+    print "\n******************************************\n";
+    print "* Testing with only curve: $curve\n";
+    print "******************************************\n";
+
+    system( "scripts/config.pl set $curve" )
+        and abort "Failed to disable $curve\n";
```

Comment:
Typo: enable

Reply:
Thanks. I amended the corresponding commit. The previous history is at https://github.com/gilles-peskine-arm/mbedtls/tree/build_with_only_montgomery_curves-conditional_mul_add-1